BreweryRatings/BusinessReview.cs

```diff
@@ -17,50 +17,66 @@ namespace ReviewNamespace
 
     public partial class BusinessReview
     {
+        private Review[] _reviews;
+        private long _total;
+        private string[] _possibleLanguage;
+
         [JsonProperty("reviews")]
-        public Review[] Reviews { get; set; }
+        public Review[] Reviews { get => _reviews; set => _reviews = value; }
```

Comment:
Encapsulation is a good idea, though {get; set; } syntax is often considered a shortcut for encapsulation.  In our class, I'm fine with either approach.